src/taxonomy/taxonomy-card/TaxonomyCardMenu.jsx

```diff
@@ -26,7 +26,10 @@ const TaxonomyCardMenu = ({
     <>
```

Comment:
I'm able to use the Export menu item -- clicking it takes me to the Taxonomy detail page.

So I think we need to run `e.preventDefault()` on the `MenuItem` onClick event below too to prevent the click event from bubbling up to the Card. And since there will be more menu items eventually, I think it's best to do it in the `onClickItem` method above, e.g

```javascript
const onClickItem = (e, menuName) => {
    e.preventDefault();
    setMenuIsOpen(false);
    onClickMenuItem(menuName);
  };
```

invoked below like this:

```javascript
<MenuItem className="taxonomy-menu-item" onClick={(e) => onClickItem(e, 'export')}>
```